Automattic/jetpack
# Social: Add recommendation card for the advanced plan

This change adds an onboarding flow for the Social Advanced plan, to make it easier for users to see our value and use our features.

## Proposed changes:

* Added three new cards
  * One for the welcome, that's presented first if they buy the advanced plan for the site `recommendations/welcome-social-advanced` 
  * There is the general card that can be revisited anytime from the recommendations screen `recommendations/-social-advanced-activated`
  * One for SIG, which is also presented in the onboarding flow after the welcome one`recommendations/welcome-social-image-generator`

### Other information:


## Jetpack product discussion


https://github.com/orgs/Automattic/projects/733/views/2?pane=issue&itemId=50248339
## Does this pull request change what data or activity we track or use?


No.
## Testing instructions:






* Create a new JN site, or delete jetpack options on your local site to reset the onboarding flows. `jetpack docker wp option delete jetpack_options`. You'll have to reconnect Jetpack after this.
* Ensure you do not have the advanced plan, then buy it for the site. You should get redirected to the Social advanced onboarding flow. There you should see 4 steps, the first one is a welcome, then SIG introduction, then a general monitor and accelerator pages.

<img src="https://github.com/Automattic/jetpack/assets/36671565/18788813-c793-4541-bb08-931b18769b37" width="45%"></img> <img src="https://github.com/Automattic/jetpack/assets/36671565/85332aa1-60fb-4bfa-bffb-5ad89ba4a4ed" width="45%"></img> 

* Make sure that the CTA-s work as intended, and then after the 4 slides you get back to the summary.
* Click `Advanced Social Sharing` there, which should direct you to the generic card.

<img src="https://github.com/Automattic/jetpack/assets/36671565/0d09d1bc-8694-4f78-9656-943e225c439b" width="45%"></img> 

Now you should test the same for the Basic plan. It will be similar, but have different slides and the Advanced social sharing row won't appear in the summary.

<img src="https://github.com/Automattic/jetpack/assets/36671565/7a9faa4b-95bf-482c-aebd-69b43bfc5af8" width="45%"></img> 



This should cover the flows for all the native Social plans.

* Now create another JN site, or reset the jetpack options to start from scratch, and now buy the Jetpack Complete bundle.
* It will direct you to the recommendations once again, and one slide should be the Social generic card from above
